src/main.cc

```diff
@@ -77,6 +77,14 @@ int wmain(int argc, const wchar_t* argv[]) {
 
       if (!updater.SetIcon(argv[++i]))
         return print_error("Unable to set icon");
+    } else if (wcscmp(argv[i], L"--set-requested-execution-level") == 0 || 
+      wcscmp(argv[i], L"-sel") == 0) {
+      if (argc - i < 2)
+        return print_error("--set-requested-execution-level requires user, requireAdministrator or asInvoker");
```

Comment:
The docs say `asInvoker|highestAvailable|requireAdministrator` so should this be `highestAvailable` here or `user`?